src/components/calcite-value-list-item/calcite-value-list-item.tsx

```diff
@@ -115,7 +115,7 @@ export class CalciteValueListItem {
   //
   // --------------------------------------------------------------------------
 
-  getPickListRef = (el): HTMLCalcitePickListItemElement => (this.pickListItem = el);
+  getPickListRef = (el: HTMLCalcitePickListItemElement) => (this.pickListItem = el);
```

Comment:
Did this change the return type?

Reply:
It should be able to infer the type, but I'll revert this change since linting still passes (I could've sworn it was failing earlier).